tools/BuildToolsetVersions.props

```diff
@@ -0,0 +1,5 @@
+<Project>
+  <PropertyGroup>
+    <BuildTasksFeedToolVersion>1.0.0-prerelease-02203-01</BuildTasksFeedToolVersion>
```

Comment:
Is this used somewhere else?

Reply:
Just in 'BuildToolset.proj'

Reply:
Eliminated 'BuildToolsetVersions.props'; 'BuildToolset.proj' holds the version